src/libraries/System.Private.CoreLib/src/System/Resources/ResourceManager.cs

```diff
@@ -98,7 +98,7 @@ internal sealed class CultureNameResourceSetPair
             public ResourceSet? lastResourceSet;
         }
 
-        protected string BaseNameField;
+        private readonly string _baseNameField;
```

Comment:
This will likely break some old .NET Framework binaries that depend on this field.  (This field was public in .NET Framework.)

I would just add comment that this protected for .NET Framework compat.